modules/services.sh

```diff
@@ -83,8 +86,37 @@ function services {
     done
   else
     if [ -z "$command" ]; then
-      echo "ERROR: no command given"
-      exit 1
+      service_list=($(services available))
+      for service in "${service_list[@]}"
+      do
+        if [ $service == $service_name ]; then
+          running_services=($(services running))
+          basic_info=$(services ${service} info)
+          echo "$basic_info"
+          echo
+          if [ -d /srv/$service ]; then
+            echo "status: installed"
+          else
+            echo "status: not installed"
+          fi
+          for i in "${running_services[@]}"
+          do
+            if [ $i == $service ]; then
+              echo "        running"
+            fi
+          done
+          autorun_status=$(services ${service} autorun)
+          echo "autorun: $autorun_status"
+          local_url=$(services ${service} url local)
+          echo "url: "$local_url | sed -e 's/ /\n     /g2'
+          tor_url=$(services ${service} url tor)
+          echo "tor: "${tor_url} | sed -e 's/ /\n     /g2'
+          port_number=$(services ${service} port)
+          echo "port: "${port_number} | sed -e 's/ /\n      /g2'
+          size=$(services ${service} size)
+          echo "size: $size"
```

Comment:
Instead of saving the outputs to a variable and then echoing that variable, you can just echo the outputs directly in one line.

For example, instead of doing:
```
autorun_status=$(services ${service} autorun)
echo "autorun: $autorun_status"
```
you can do:
```
echo "autorun: $(services ${service} autorun)"
```
or if you remove the top for loop:
```
echo "autorun: $(services $service_name autorun)"
```